packages/icons/components/TwoPeople.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FC } from 'react';
+import { Path, Svg } from 'react-native-svg';
+
+import type { IconProps } from './types';
+
+export const TwoPeople: FC<IconProps> = ({ size = 24, color = '#FFFFFF' }) => {
+	return (
+		<Svg width={size} height={size} viewBox="0 0 14 10" fill="none">
```

Comment:
this `viewBox` props show that width and height are not equal, check if dimension is correct?